react/components/EXPERIMENTAL_Table/DataTable/Cell.tsx

```diff
@@ -35,19 +35,31 @@ const Cell: FC<CellProps> & CellComposites = ({
   className: classNameProp = '',
   active = false,
   link = false,
+  sticky = false,
+  header,
```

Comment:
why is this prop necessary?

Reply:
updated.

Reply:
@matheusps why `header` is necessary?

Reply:
Ops. I edited my first answer, for some reason just 'updated' stayed 😅(sorry). The header defines if is a `th` or `td`. And we may not use `sticky` on a `td`.